include/z64ocarina.h

```diff
@@ -0,0 +1,123 @@
+#ifndef Z64OCARINA_H
+#define Z64OCARINA_H
+
+#include "ultra64.h"
+
+typedef enum {
+    /*  0 */ OCARINA_SONG_SONATA,
+    /*  1 */ OCARINA_SONG_GORON_LULLABY,
+    /*  2 */ OCARINA_SONG_NEW_WAVE,
+    /*  3 */ OCARINA_SONG_ELEGY,
+    /*  4 */ OCARINA_SONG_OATH,
+    /*  5 */ OCARINA_SONG_SARIAS,
+    /*  6 */ OCARINA_SONG_TIME,
+    /*  7 */ OCARINA_SONG_HEALING,
+    /*  8 */ OCARINA_SONG_EPONAS,
+    /*  9 */ OCARINA_SONG_SOARING,
+    /* 10 */ OCARINA_SONG_STORMS,
+    /* 11 */ OCARINA_SONG_SUNS,
+    /* 12 */ OCARINA_SONG_INVERTED_TIME,
+    /* 13 */ OCARINA_SONG_DOUBLE_TIME,
+    /* 14 */ OCARINA_SONG_GORON_LULLABY_INTRO,
+    /* 15 */ OCARINA_SONG_WIND_FISH_HUMAN, // "Ballad of the Wind Fish"
+    /* 16 */ OCARINA_SONG_WIND_FISH_GORON,
+    /* 17 */ OCARINA_SONG_WIND_FISH_ZORA,
+    /* 18 */ OCARINA_SONG_WIND_FISH_DEKU,
+    /* 19 */ OCARINA_SONG_EVAN_PART1,
+    /* 20 */ OCARINA_SONG_EVAN_PART2,
+    /* 21 */ OCARINA_SONG_ZELDAS_LULLABY,
+    /* 22 */ OCARINA_SONG_SCARECROW,
+    /* 23 */ OCARINA_SONG_TERMINA_WALL,
+    /* 24 */ OCARINA_SONG_MAX,
+    /* 24 */ OCARINA_SONG_SCARECROW_LONG = OCARINA_SONG_MAX // anything larger than 13 is considered the long scarecrow's song
```

Comment:
What do you mean by "anything larger than 13"?  I can only see a 24 here

Reply:
The `13` was a result of copying from OoT and should be 24, my bad haha!
https://github.com/zeldaret/oot/blob/0b8edc21c03a719d8f7874bc20204612d6d1c107/include/z64.h#L348

This message seems to comes from `z_message` and how ocarina songs are processed. I can take this comment out until `z_message` in MM is decompiled to see if it has the same behaviour